components/brave_wallet/browser/internal/hd_key_utils.h

```diff
@@ -0,0 +1,56 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_WALLET_BROWSER_INTERNAL_HD_KEY_UTILS_H_
+#define BRAVE_COMPONENTS_BRAVE_WALLET_BROWSER_INTERNAL_HD_KEY_UTILS_H_
+
+#include <optional>
+#include <string_view>
+#include <vector>
+
+#include "base/containers/span.h"
+
+namespace brave_wallet {
+
+namespace internal {
+void SecureZeroBuffer(base::span<uint8_t> data);
```

Comment:
Seems like this would make more sense as a static private method on `ScopedSecureZeroSpan` and use friend for `SecureByteArray` and tests?

Reply:
actually this method is just calling `crypto::internal::SecureZeroBuffer` so we don't need it at all

Reply:
That was to avoid exposing `crypto/process_bound_string.h` through header.
Now removed